internal/sirius/case.go

```diff
@@ -0,0 +1,37 @@
+package sirius
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type Case struct {
+	UID      string `json:"uId"`
+	CaseType string `json:"caseType"`
+}
+
+func (c *Client) Case(ctx Context, id int) (Case, error) {
+	var v Case
+
+	req, err := c.newRequest(ctx, http.MethodGet, fmt.Sprintf("/api/v1/cases/%d", id), nil)
+	if err != nil {
+		return v, err
+	}
+
+	res, err := c.http.Do(req)
+	if err != nil {
+		return v, err
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
+		return v, newStatusError(res)
+	}
+
+	if err := json.NewDecoder(res.Body).Decode(&v); err != nil {
+		return v, err
+	}
```

Comment:
It'd be nice to move some of this to a shared function at some point, it's a lot of boilerplate for what could just be `return c.get(fmt.Sprintf("/api/v1/cases/%d", id))`

Reply:
probably true for POSTs too, I think I'll leave for next time